src/util/parse_options.h

```diff
@@ -34,12 +36,15 @@ class parse_options_baset
   virtual int main();
   virtual ~parse_options_baset() { }
 
+private:
+  bool parse_result;
+
 protected:
+  ui_message_handlert message_handler;
```

Comment:
I think you could reduce the size of the diff quite a bit by just naming this `ui_message_handler`.

Reply:
I also have a slight preference for `ui_message_handler` in order to distinguish it from a plain message_handler.